src/page/basicBrowserFeatureCheck.js

```diff
@@ -76,5 +76,7 @@ function isMinimumSupported() {
 }
 
 if (!isMinimumSupported()) {
+  // eslint-disable-next-line no-console
+  console.error('The minimum browser features are not supported to run the Wire app!');
```

Comment:
I don't have a lot of insight on that unsupported browser subject.
But is this a log we intend to keep in production or is it just for testing purposes?

Reply:
I would like it on production, not just for testing purposes. Imagine we get some bug about an unsupported browser, and if a user can provide us with this log we will be able to immediately know where to look for the issues. In any case, this log will be almost the only one in the console, as the app won't run, so it should not spam it

Reply:
Also, since we removed anything technical from the text shown to the user, someone technical will still see that there is a problem with cookies or with IndexDB. There is a very low probability it is needed, but it is still nice to have

Reply:
id also lean towards another solution as console logs arent really something we tend to do in prod.

Reply:
Any ideas how else we can implement this? Or you prefer to remove it completely? I'm against console logs for the application too, but this is even the code that is not a part of the main app codebase. When we are showing just one screen "Your browser is not supported" without anything technical about cookies or IndexDB (private mode), I thought it was ok to throw an error into the console.